.github/workflows/test_scripts.yml

```diff
@@ -19,6 +19,7 @@ jobs:
             python-cleanliness,
             astyle-cleanliness,
             validate_board_list,
+            pre-commit-cleanliness,
```

Comment:
Would need an entry in `Tools/scripts/builds_ci.sh` IIUC

Reply:
Fixed, accidentally dropped it during the rebase. It was originally there on this commit. 
https://github.com/ArduPilot/ardupilot/commit/48cc7b110787984eb86527cd81e4f0028b98ee61#diff-1600157c5dce475648f144b0f9b010395f5a9d5ad66a3fe9e56a3508cd930c67